drivers/bluetooth/hci/hci_spi_st.c

```diff
@@ -398,6 +399,26 @@ static struct net_buf *bt_spi_rx_buf_construct(uint8_t *msg)
 		}
 		net_buf_add_mem(buf, &msg[1], len);
 		break;
+#if defined(CONFIG_BT_ISO)
+	case HCI_ISO:
+		struct bt_hci_iso_hdr iso_hdr;
+
+		buf = bt_buf_get_rx(BT_BUF_ISO_IN, timeout);
+		if (buf) {
+			memcpy(&iso_hdr, &msg[1], sizeof(iso_hdr));
+			len = sizeof(iso_hdr) + bt_iso_hdr_len(sys_le16_to_cpu(iso_hdr.len));
+		} else {
+			LOG_ERR("No available ISO buffers!");
+			return NULL;
+		}
```

Comment:
I've experienced that logging each missing ISO buffer can effectively use the entire CPU if they come frequent enough, and have suggested https://github.com/zephyrproject-rtos/zephyr/pull/69867 for the IPC driver to avoid this issue. 

Worth considering applying something similar here

Reply:
wouldn't it be better to just `LOG_DBG`?

Reply:
Linux has dedicated `printk_ratelimit()` API for this kind of situations - might be worth considering if something like that would make sense in the Zephyr logging API.

Reply:
I would prefer to keep the code as it is since I've not faced such a problem until a proper solution is introduced in Zephyr. What's your opinion?

Reply:
Feel free to keep as is (not a change request)